### PR TITLE
feat: Added EmojiRegex

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: Skyra.Tests/Core/Utils/EmojiRegexTests.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+using NUnit.Framework;
+using Skyra.Core.Utils;
+
+namespace Skyra.Tests.Core.Utils
+{
+	public class EmojiRegexTests
+	{
+		[Test]
+		public void EmojiRegex_Match([Values(
+				"😀", "😃", "😄", "😁", "😆", "😅", "🤣", "😂", "🙂", "🙃", "😉", "😊", "😇", "🥰", "😍", "🤩", "😘",
```
Comment (Tylertron1998) [inline Skyra.Tests/Core/Utils/EmojiRegexTests.cs:12]: might be better to put this into a readonly string[] at the top.

File: Skyra.Tests/Core/Utils/EmojiRegexTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Skyra.Core.Utils;
+
+namespace Skyra.Tests.Core.Utils
+{
+	public class EmojiRegexTests
+	{
+		[Test]
+		public void EmojiRegex_Match([Values(
```
Comment (Tylertron1998) [inline Skyra.Tests/Core/Utils/EmojiRegexTests.cs]: suggestion: more descriptive test name, `EmojiRegex_Matches_ValidEmojis`

File: Skyra.Tests/Core/Utils/EmojiRegexTests.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+			)]
+			[NotNull]
+			string emoji
+		)
+		{
+			Assert.IsTrue(EmojiRegex.Regex.IsMatch(emoji));
+		}
+
+		[Test]
+		public void EmojiRegex_NotMatch([Values(
```
Comment (Tylertron1998) [inline Skyra.Tests/Core/Utils/EmojiRegexTests.cs]: same thing here, `EmojiRegex_DoesntMatch_InvalidEmojis`

File: Skyra.Tests/Core/Utils/EmojiRegexTests.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Skyra.Core.Utils;
+
+namespace Skyra.Tests.Core.Utils
+{
+	public class EmojiRegexTests
+	{
+		[Test]
```
Comment (Tylertron1998) [inline Skyra.Tests/Core/Utils/EmojiRegexTests.cs:9]: Adding `[Parallelizable]` to both tests will allow the test to run in... well, parallel (who'd have thought?)

File: Skyra.Tests/Core/Utils/EmojiRegexTests.cs (hunk 1 above)
Comment (kyranet) [reply Skyra.Tests/Core/Utils/EmojiRegexTests.cs:12]: Not possible, needs to be a constant.

**Pull request merged**